src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/infrastruktur/MockServer.java

```diff
@@ -4,11 +4,11 @@
 import com.github.tomakehurst.wiremock.client.WireMock;
 import com.github.tomakehurst.wiremock.common.ConsoleNotifier;
 import com.github.tomakehurst.wiremock.core.WireMockConfiguration;
-import com.github.tomakehurst.wiremock.matching.UrlPathPattern;
-import com.github.tomakehurst.wiremock.matching.UrlPattern;
+import com.github.tomakehurst.wiremock.matching.*;
```

Comment:
Jeg tror at det er IntelliJ som bytter til `.*` i import, ble vi ikke enige om at vi bruker heller single import by class, mtp sikkerhet osv.?

Reply:
Det er en setting som du kan endre i IntelliJ: https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports

Reply:
Fixed